packages/arb-bridge-eth/test/test_cases.json

```diff
@@ -5,18 +5,6 @@
     "hash": "c65a7bb8d6351c1cf70c95a316cc6a92839c986682d98bc35f958f4883f9d2a8",
     "name": "int"
   },
```

Comment:
would be great to have all these generated automatically at some point

Reply:
I actually shifted over to auto-replacing all of these in the PR that will come in after this one. We'll have to decided how we want to handle this whole thing once we dump the python compiler since that's how these are generated now. We'll probably want to keep the test cases idea, but generate it from a different codebase